experimental/AlgebraicStatistics/src/GaussianGraphicalModels.jl

```diff
@@ -0,0 +1,454 @@
+###################################################################################
+#
+#       Gaussian Rings
+#
+###################################################################################
+
+export GaussianRing
+export gaussian_ring, ring, gens, covariance_matrix
+export graphical_model, parameterization
+export directed_edges_matrix, error_covariance_matrix
+export concentration_matrix
+
+struct GaussianRing
+    ring::Ring
+    gens::Dict
+    covariance_matrix
+end
+
+
+
+@doc raw"""
+    gaussian_ring(n::Int; s_var_name::String="s", base_ring=QQ)
+
+A polynomial ring whose variables correspond to the entries of a covariance matrix of `n` Gaussian random variables.
+It is a multivariate polynomial ring whose variables are named `s[i,j]`and whose `base_ring` is by default `QQ`. 
+
+## Examples
+
+``` jldoctest
+julia> R = gaussian_ring(3)
+Gaussian ring over Rational field in 6 variables
+s[1, 1], s[1, 2], s[1, 3], s[2, 2], s[2, 3], s[3, 3]
+```
+"""
+function gaussian_ring(n::Int, s_var_name::String="s", base_ring::Field=QQ)
+
+    varindices = [Tuple([i,j]) for i in 1:n for j in i:n]
+    varnames = ["$(s_var_name)[$(i), $(j)]" for (i,j) in varindices]
+    S, s = polynomial_ring(base_ring, varnames)
```

Comment:
Probably should add `cached=false` to most/all `polynomial_ring` invocations, see also https://docs.oscar-system.org/dev/DeveloperDocumentation/caching/#Rules-for-implementations (and a bunch of recent PRs which add missing `cached=false` args in many places)

Reply:
Done.